docs/_sources/installation.rst.txt

```diff
@@ -9,53 +9,57 @@ The following section includes installation instructions for `MHKiT-Python <http
 MHKiT-Python
 -------------
 
-`MHKiT-Python <https://github.com/MHKiT-Software/MHKiT-Python>`_ requires `Python (3.6 or 3.7) <https://www.python.org/>`_ , and has several Python package dependencies.  
-It is recommended to use the `Anaconda Python Distribution <https://www.anaconda.com/distribution/>`_ to install Python, since it includes most of MHKiT-Python's package dependencies.
-Refer to the `Python website <https://www.python.org/>`_ for information on using Python.  
+`MHKiT-Python <https://github.com/MHKiT-Software/MHKiT-Python>`_ requires `Python (3.6 or 3.7) <https://www.python.org/>`_  and has several Python package dependencies.
+It is recommended to use the `Anaconda Python Distribution <https://www.anaconda.com/distribution/>`_ to install Python since it includes most of MHKiT-Python's package dependencies.
+Refer to the `Python website <https://www.python.org/>`_ for information on using Python.
 
+.. Note::
+    Anaconda3 no longer adds python to the Windows Environment Variables by default.There is a check box in the advanced options of the installation to add it to the system path, but the GUI advises against so.
```

Comment:
How about?
During installation Anaconda3 no longer adds python to the Windows Environment Variables by default. During installation, there is a checkbox in the advanced options of the installation to add it to the system path. The installation wizard advises against doing so in case a user has multiple python installations. Either check this box or setup your environmental variables so that python is accessible through your system path.